python/kfserving/kfserving/storage.py

```diff
@@ -150,15 +149,15 @@ def _download_blob(uri, out_dir: str):  # pylint: disable=too-many-locals
                      container_name,
                      prefix)
         try:
-            block_blob_service = BlockBlobService(account_name=account_name)
-            blobs = block_blob_service.list_blobs(container_name, prefix=prefix)
-        except Exception:  # pylint: disable=broad-except
+            connect_str = os.getenv('AZURE_STORAGE_CONNECTION_STRING')
```

Comment:
how is this environment variable going to be set?

Reply:
Good question. The connection string, is highlighted in the documentation, for best practices.

https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-python

However the storage initializer sidecar will not have a way to provide this information or a way to support it. Do we want to add support here? I think that would be the best approach. 

https://github.com/kubeflow/kfserving/blob/master/docs/samples/storage/azure/README.md
https://github.com/kubeflow/kfserving/blob/42007c532286a1e43893ef2be03b15e104bfd7a4/pkg/credentials/azure/azure_secret.go

Let me know your thoughts.